modules/ROOT/pages/policies-policy-categories.adoc

```diff
@@ -0,0 +1,39 @@
+= Policy Categories
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are categorized by the funtion they perform. For example, becuase you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a Troubleshooting policy. The following table lists the Default policy name, its category and the function that it performs. +
+You can apply only one policy of each category at a given time.
+
+[%header,cols="40a,15a,60a"]
+|===
+| Policy | Category | Function
+| Basic Authentication - LDAP | Security | Authenticates the LDAP credentials.
+| Basic Authentication - Simple | Security | Authenticates single user password.
+| Client ID Enforcement | Compliance | Allows access to client applications with a valid Client ID.
+| CORS | Compliance | Enables calls executed in a web page to interact with resources from non-origin domains.
+| Header Injection | Transformation | Adds headers to the request or response message of a policy.
+| Header Removal | Transformation | Removes headers from the request or response message of a policy.
+| HTTP Caching | Quality of Service | Stores HTTP responses from an API implementation.
+| IP Blacklist | Security | Blocks a range of IP addresses.
+| IP Whitelist | Security | Allows acces from only from a lpre-approved range of IP addresses.
+| JSON Threat Protection | Security | Protects against malicious JSON structure in API requests.
+| Message Logging | Troubleshooting | Logs a custom message when an API is invoked.
+| OAuth 2.0 Access Token Enforcement Using External Provider Policy | Security | OAuth 2.0 protected
+| OpenAM Access Token Enforcement | Security | Restricts access to a protected resource using an Opem AM authentication server.
+| PingFederate Access Token Enforcement | Restricts access to a protected resource using PingFederate authentication.
```

Comment:
The Security category is missing

Reply:
"Ping Federate authentication" replace with "PingFederate authorization server"

Reply:
@evangelina what do you mean by Security category is missing?

Reply:
Got it! Great catch! Fixed!